daemon/inertiad/auth/permissions.go

```diff
@@ -154,8 +165,11 @@ func (h *PermissionsHandler) ServeHTTP(w http.ResponseWriter, r *http.Request) {
 		}
 	}
 
+	// Attach token to request context so handlers can use it
+	ctx := context.WithValue(r.Context(), ctxUsername, claims.User)
```

Comment:
the comment here should be `Attach user to request context` right?

Reply:
Yea.

Reply:
Done.